apps/mobile/src/components/ChatDetailScreen/components/QuickActionBanner.tsx

```diff
@@ -311,7 +310,13 @@ function QuickActionBanner(): JSX.Element | null {
     )
   }
 
-  if (contactRevealStatus === 'shared') {
+  if (
+    (lastMessage.message.messageType === 'APPROVE_CONTACT_REVEAL' ||
+      (lastMessage.message.messageType === 'TRADE_CHECKLIST_UPDATE' &&
+        lastMessage.message.tradeChecklistUpdate?.contact?.status ===
+          'APPROVE_REVEAL')) &&
```

Comment:
We want to show this only if the last message is response on reveal? Is that desired? If I approve contact reveal and I send text  message right after that, will the other side see the banner? Should it?